extension/src/test/suite/experiments/workspace.test.ts

```diff
@@ -561,57 +563,66 @@ suite('Workspace Experiments Test Suite', () => {
           {
             description:
               '$(git-commit)Update version and CHANGELOG for release (#4022) ...',
-            detail: 'loss:2.0488560, accuracy:0.34848332, val_loss:1.9979370',
+            detail:
+              'code_names:[0,1], epochs:5, learning_rate:2.1e-7, loss:2.0488560, accuracy:0.34848332, val_loss:1.9979370',
             label: 'main',
             value: 'main'
           },
           {
             description: '[exp-e7a67]',
-            detail: 'loss:2.0205045, accuracy:0.37241668, val_loss:1.9979371',
+            detail:
+              'code_names:[0,1], epochs:2, learning_rate:2e-12, loss:2.0205045, accuracy:0.37241668, val_loss:1.9979371',
```

Comment:
Is using six column values too lengthy for the quick pick? 

<img width="577" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/71bd6967-89f4-4ed7-8fb2-1f04e3c4648e">

Reply:
No, I think it is fine.